stories/index.js

```diff
@@ -41,7 +41,40 @@ storiesOf('Picker', module)
       showPreview={boolean('Show preview', true)}
       showSkinTones={boolean('Show skin tones', true)}
       custom={CUSTOM_EMOJIS}
-    />
+    />))
+    .add('custom-svg', () => (
```

Comment:
More idiomatic for react storybook would be 'with custom icons' (instead of 'custom-svg')